### DISCUSSION TITLE
Bi-Weekly Community Meeting, Saturday, June 1, 2024

### DISCUSSION THREAD

**Discussion by wrashi** [GothenburgBitFactory/taskwarrior#3464] [Announcements]
The TaskWarrior community bi-weekly is coming up!
Check out the [conversation which launched our first meeting](https://github.com/GothenburgBitFactory/taskwarrior/discussions/3403). 
Or watch the meeting videos…
- [May 4](https://youtu.be/gcZ6MZYA_88)
- [May 18](https://www.youtube.com/watch?v=r5G9yiDrgkg)

Our next meeting is 12:00 UTC Saturday, June 1, 2024 ([check that in your time zone @ timeanddate.com](https://www.timeanddate.com/worldclock/meetingtime.html?month=6&day=1&year=2024&p1=5&p2=3904&p3=3918&p4=3921&iv=0)). You can [add this event to your calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NWpqbTNrbmNzYmxycjNzdjBnNzZxcGNwY2ggd2VzbGV5QG13cGVubmVyLmNvbQ&tmsrc=wesley%40mwpenner.com).

We'll copy last meeting's success and use Google Meet:
Video call link: https://meet.google.com/tns-suqq-iaj  
Or dial: ‪(CA) +1 289-323-1423‬ PIN: ‪830 535 232‬#  
More phone numbers: https://tel.meet/tns-suqq-iaj?pin=8726596308824

The formal agenda portion of the meeting ~30 minutes. The room will stay open for anyone who wants to hang out or go deeper into any of the topics from the agenda.

**Comment by wrashi** [comment]
Saturday's meeting was nice. Dustin showed his TaskWarrior workflow - 15 years of use and improvement really shows! Lots of other great conversation, too! Check it out here…

https://youtu.be/hZQG8jDn24I

**Comment by ryneeverett** [comment]
Dustin's Gist: https://gist.github.com/djmitche/dd7c9f257306e6b8957759c4d5265cc9

**Reply by wrashi** [reply]
Thanks, @ryneeverett! I did add that in the video comments when I noticed my oversight, but should have been more on top of it here, too!

**Comment by ryneeverett** [comment]
@djmitche Thanks for clarifying what GTD means to you. I'm relieved to not be the only one who hasn't read the GTD book. :)

**Reply by djmitche** [reply]
No problem! Hopefully no GTD purists try to hold me to the details.
